mobile/lib/repositories/album.repository.dart

```diff
@@ -43,14 +44,11 @@ class AlbumRepository extends DatabaseRepository implements IAlbumRepository {
     if (shared != null) {
       query = query.sharedEqualTo(shared);
     }
+    final userId = fastHash(Store.get(StoreKey.currentUser).id);
```

Comment:
Can we change any variable name that use `fastHash` method to `isarUserId` to distinguish it from other UUID?